From 995efe4d44ebb97380d12130f329e364b9d7da7e Mon Sep 17 00:00:00 2001
From: Andreas Schmidt <andreas.schmidt@wago.com>
Date: Tue, 25 Sep 2018 13:47:51 +0200
Subject: [PATCH] dts: 750_8215: set eee broken for internal phys

Internal PHYs of MV88E6321 have EEE enabled by default.
But EEE in combination with e.g. Cut Through not reliable work.
This patch disable (mark as broken) EEE for internal PHYs.

Signed-off-by: Andreas Schmidt <andreas.schmidt@wago.com>
---
 arch/arm/boot/dts/am335x-pfc-750_8215.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-pfc-750_8215.dts b/arch/arm/boot/dts/am335x-pfc-750_8215.dts
index 649602e..1ed406f 100644
--- a/arch/arm/boot/dts/am335x-pfc-750_8215.dts
+++ b/arch/arm/boot/dts/am335x-pfc-750_8215.dts
@@ -175,6 +175,8 @@
 				max-speed = <100>;
 				interrupt-parent = <&switch>;
 				interrupts = <3 IRQ_TYPE_LEVEL_HIGH>;
+				eee-broken-100tx;
+				eee-broken-1000t;
 			};
 
 			int_phy4: ethernet-phy@4 {
@@ -183,6 +185,8 @@
 				max-speed = <100>;
 				interrupt-parent = <&switch>;
 				interrupts = <4 IRQ_TYPE_LEVEL_HIGH>;
+				eee-broken-100tx;
+				eee-broken-1000t;
 			};
 		};
 
-- 
2.7.4

